pkg/controller/proto/meta.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto3";
```

Comment:
How do we generate the `pb.go` files when the proto file changes? Please make it reproducible by either using `go:generate` or creating a target in the makefile.

Reply:
I Agee. I added it in the Makefile